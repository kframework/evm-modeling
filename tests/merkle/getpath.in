#getPath( #HPEncode( #nibbleize( "asdf" ), 1 ) )
