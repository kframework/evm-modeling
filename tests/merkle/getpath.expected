<k>
  b"\x06\x01\x07\x03\x06\x04\x06\x06"
</k>
